Sources/Defaults/Defaults+Bridge.swift

```diff
@@ -407,15 +412,41 @@ extension Defaults {
 				return nil
 			}
 
-			return NativeColor.bridge.serialize(NativeColor(value))
+			guard
+				let cgColor = value.cgColor,
+				let colorSpace = cgColor.colorSpace?.name as? String,
+				let components = cgColor.components
+			else {
+				return NativeColor.bridge.serialize(NativeColor(value))
+			}
+
+			return [colorSpace, components]
 		}
 
 		public func deserialize(_ object: Serializable?) -> Value? {
-			guard let nativeColor = NativeColor.bridge.deserialize(object) else {
+			if let object = object as? NativeColor.Serializable {
+				guard let nativeColor = NativeColor.bridge.deserialize(object) else {
+					return nil
+				}
+
+				return Value(nativeColor)
+			}
+
+			guard
+				let object = object as? [Any],
+				let rawColorspace = object[0] as? String,
+				let colorspace = CGColorSpace(name: rawColorspace as CFString),
+				let components = object[1] as? [CGFloat],
+				let cgColor = CGColor(colorSpace: colorspace, components: components)
+			else {
 				return nil
 			}
 
-			return Value(nativeColor)
+			#if os(macOS)
+			return Value(cgColor)
+			#else
+			return Value(cgColor: cgColor)
+			#endif
```

Comment:
Nitpick, but it would be better to use an available check:

```suggestion
			if #available(iOS 15, macOS 12, tvOS 15, watchOS 8, *) {
				return Value(cgColor: cgColor)
			} else {
				return Value(cgColor)
			}
```